op-challenger/game/fault/validator.go

```diff
@@ -40,7 +42,7 @@ func (v *PrestateValidator) Validate(ctx context.Context) error {
 		return fmt.Errorf("failed to fetch provider's prestate hash: %w", err)
 	}
 	if !bytes.Equal(prestateCommitment[:], prestateHash[:]) {
-		return fmt.Errorf("provider's absolute prestate does not match contract's absolute prestate: Provider: %s | Contract: %s", prestateCommitment.Hex(), prestateHash.Hex())
+		return fmt.Errorf("provider's %v absolute prestate does not match contract's absolute prestate: Provider: %s | Contract: %s", v.valueName, prestateCommitment.Hex(), prestateHash.Hex())
```

Comment:
The `valueName` is passed in as `alphabet`, `cannon`, and `output root`. Should this instead be

```suggestion
		return fmt.Errorf("%v provider's absolute prestate does not match contract's absolute prestate: Provider: %s | Contract: %s", v.valueName, prestateCommitment.Hex(), prestateHash.Hex())
```

Reply:
Both ways work depending on how you think about it.  If the trace provider is the combined one then `provider's cannon absolute prestate` is right and if the provider is the nested one then `cannon provider's absolute prestate` is right.

Reply:
Have switched it round though.